sdk/search/search-documents/README.md

```diff
@@ -84,39 +81,122 @@ const serviceClient = new SearchServiceClient("<endpoint>", new AzureKeyCredenti
+-   [Indexer operations](https://docs.microsoft.com/en-us/rest/api/searchservice/indexer-operations). Automate aspects of an indexing operation by configuring a data source and an indexer that you can schedule or run on demand. This feature is supported for a limited number of data source types.
+-   [Skillset operations](https://docs.microsoft.com/en-us/rest/api/searchservice/skillset-operations). Part of a cognitive search workload, a skillset defines a series of a series of enrichment processing steps. A skillset is consumed by an indexer.
+-   [Synonym map operations](https://docs.microsoft.com/en-us/rest/api/searchservice/synonym-map-operations). A synonym map is a service-level resource that contains user-defined synonyms. This resource is maintained independently from search indexes. Once uploaded, you can point any searchable field to the synonym map (one per field).
 
-`SearchServiceClient` provides methods for configuring and customizing an Azure Cognitive Search instance. The client currently has support for creating and managing search indexes and will later expand to support creating and managing other service entities such as indexers, synonym maps, cognitive skillsets, and data sources.
+## Examples
 
-**Note**: This client cannot function in the browser because the APIs it calls do not have support for Cross-Origin Resource Sharing (CORS).
+### Create an Index
 
-### Documents
+```js
+const { SearchServiceClient, AzureKeyCredential } = require("@azure/search-documents");
 
-An item stored inside a search index. The shape of this document is described in the index using `Field`s. Each Field has a name, a datatype, and additional metadata such as if it is searchable or filterable.
+const client = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
 
-### Pagination
+async function main() {
+  const result = await client.createIndex({
+    name: "example-index",
+    fields: [
+      {
+        type: "Edm.String",
+        name: "id",
+        key: true
+      },
+      {
+        type: "Edm.Double",
+        name: "awesomenessLevel",
+        sortable: true,
+        filterable: true,
+        facetable: true
+      },
+      {
+        type: "Edm.String",
+        name: "description",
+        searchable: true
+      },
+      {
+        type: "Edm.ComplexType",
+        name: "details",
+        fields: [
+          {
+            type: "Collection(Edm.String)",
+            name: "tags",
+            searchable: true
+          }
+        ]
+      },
+      {
+        type: "Edm.Int32",
+        name: "hiddenWeight",
+        hidden: true
+      }
+    ]
+  });
 
-Typically you will only wish to [show a subset of search results](https://docs.microsoft.com/azure/search/search-pagination-page-layout#total-hits-and-page-counts) to a user at one time. To support this, you can use the `top`, `skip` and `includeTotalResultCount` parameters to provide a paged experience on top of search results.
```

Comment:
I also think talking about pagination is important, since unlike many other client libraries, client side paging is the preferred model

Reply:
I have added another section ```Typescript/Javascript specific concepts``` and added this information back

Reply:
Interesting point -- reminds me of what Ted Glava made the other day about client vs server focus.  Client SDKs might want to surface client-specific examples, perhaps in addition to (or instead of) the core create/load index workflow:  pagination, retrieving a specific page, facets, filters.  You could use the hosted NYC jobs index on a sandbox service to save steps (see the .NET SDK readme for an example)